src/prommis/solvent_extraction/solvent_extraction.py

```diff
@@ -80,8 +80,46 @@
     useDefault,
 )
 from idaes.core.util.config import is_physical_parameter_block
+from idaes.core.initialization import ModularInitializerBase
 from idaes.models.unit_models.mscontactor import MSContactor
 
+
+class SolventExtractionInitializer(ModularInitializerBase):
+    """
+    This is a general purpose Initializer  for the Solvent Extraction unit model.
+
+    This routine calls the initializer for the internal MSContactor model.
+
+    """
+
+    CONFIG = ModularInitializerBase.CONFIG()
+
+    def initialize_main_model(
+        self,
+        model: Block,
+    ):
+        """
+        Initialization routine for MSContactor Blocks.
+
+        Args:
+            model: model to be initialized
+
+        Returns:
+            None
+        """
+        model.mscontactor.material_transfer_term.fix(1e-8)
+
+        msc_init = model.mscontactor.default_initializer()
```

Comment:
You should use `get_submodel_initializer` here. See line 602 in https://github.com/IDAES/idaes-pse/blob/main/idaes/core/initialization/initializer_base.py.